### ISSUE TITLE
Peer review by Zixin Huang

### ISSUE THREAD

**Issue by zxhuang96** [SZ559/Analysis-of-Learning-Rate-for-Deep-Neural-Networks#1]
The choice of the learning rate is a detailed but important problem in machine learning, especially in heavy models like deep neural networks. Using the appropriate learning rate will enable us to train the model efficiently and improve the model’s performance. This project aims to find an effective way to identify the optimal learning rate for deep neural networks, as well as to analyze and make improvements on existing algorithms. This project will involve using various kinds of data, such as image data for image recognition, face recognition, object detection, etc.

This project is very meaningful and the results are likely to be highly useful. First, the project focuses on a widespread and crucial problem in training deep neural networks, and the algorithms proposed can be used extensively in various neural networks. Second, the algorithms will save a lot of time and improve the performance significantly at the same time. Third, the group members have prior experience in deep neural networks, which they can leverage to evaluate whether the algorithms are practical and effective. In addition, the group has made a clear plan for every stage of the project. 

Here are several suggestions that I hope can be useful for the group: 
1. Deep neural networks and image data sets may involve large scales of computation, so to guarantee the working efficiency, it could be helpful to find a high-performance computer or reasonably reduce the data size.
2. The choice of the learning rate actually depends largely on the data used, so I think it is advisable for the group to discuss whether the algorithms will be efficient and effective for all kinds of data sets, including very large data sets, heavily skewed data sets, data sets with missing values, etc. 
3. The choice of the learning rate is a widespread problem for many kinds of machine learning models, which include but not limited to deep neural networks, so it would be useful if the project can discuss whether the algorithms can be potentially applied in other types of models with or without reasonable adjustments.

**Comment by YijiaCui** [comment]
This group’s project is about deep neural networks and methods to improve their adaptive learning rate. They plans to study and analyze the existing algorithms for adaptive learning rate for Deep Neural networks, and to come up with a new method to identify an optimal learning rate and to improve the performance of existing algorithm. After they finish improving the algorithms, they will use several dataset to validate the reliability of the improved algorithms. 

Three points that I like about this proposal:
1. It is very professionally written. 
2. They have done previous work for this project. They read paper about the topics, and both of them have experience working on neural networks.
3. They have clear timeline.

Three things that can be improved:
1. It would be better if they provide an example of dataset that they are going to use to test the reliability.
2. For the significance part, it would be better if they can provide some example of how improved neural network will perform better in different fields. The topic is awesome, but it can be a little hard to understand for people who have never heard of neural network.
3. This proposal is so well written that I cannot come up with the third point of improvement. But in general, more examples and explanations of the concepts will be helpful.
